2-core-rules.md

```diff
@@ -214,108 +209,97 @@ _Fleeing_ models must take a _leadership test_. If the test is passed, the model
 
 #### Charging more than one opponent
 
 If you can move your warrior into base contact with more than one enemy model with its charge move, it can charge them both. This might be inadvisable as it'll then be fighting two enemies at once!
 
 #### Hiding
 
-The Hiding rule represents warriors concealing themselves in a way that our unmoving and dramatically posed models cannot. A hiding warrior keeps as still as possible, just peeking out of cover. 
-
-A model can hide if he ends his move behind a low wall, a column or in a similar position where he could reasonably conceal himself. The player must declare that the warrior is hiding and place a Hidden counter beside the model for it to count as being hidden. 
+The _Hiding_ rule represents warriors concealing themselves in a way that our unmoving and dramatically posed models cannot. A _Hiding_ warrior conceals themself as best they can by stiffening alongside a column, crouching behind a small wall, or hiding inside a barrel. A model can hide if they end their move behind a low wall, a column or in a similar position where they could reasonably conceal themself. The player must declare that the warrior is hiding for it to count as being _Hidden_. It may be worth placing a counter next to it so that other players are aware of it. 
 
 \page
 
-A model that runs, flees, is _stunned_ or charges cannot hide that turn. His sudden burst of speed does not give him time to hide.
+A model that is _Hidden_ is disregarded by the enemy warband's mandatory decisions. For example, a _Hidden_ model will not prevent an enemy warband from running, will not count as the closest model for the purpose of shooting, and _Frenzied_ models will not have to charge it. It is also considered wholly out of line of sight if an opponent wishes to charge them.
+
+A model that runs, is _incapacitated_, or charges cannot hide that turn. Their sudden burst of speed does not give them time to hide. They also may not enter the _Hidden_ mode if there are enemies within 4", or if they are within unobstructed line of sight to an enemy.
 
-A model may stay hidden over several turns, so long as he stays behind a wall or similar feature. He may even move around so long as he stays hidden while doing so. If an enemy moves so that he can see the hidden warrior, the model is no longer hidden and the counter is removed. When hidden, a warrior cannot be seen, shot at or charged.
+A model may stay hidden over several turns, so long as they stay behind a wall or similar feature. They may even move around so long as they stay hidden while doing so. 
 
-While hiding, a model cannot shoot or cast spells without giving away its position. If a hidden model shoots, or moves so that he can be seen, he is no longer hidden and can be shot at as normal.
+While hiding, a model cannot shoot or cast spells without giving away its position. If a hidden model shoots, or moves so that the can be seen, he is no longer hidden and can be shot at as normal.
```

Comment:
```suggestion
While hiding, a model cannot shoot or cast spells without giving away its position. If a hidden model shoots, or moves so that they can be seen, they are no longer hidden and can be shot at as normal.
```

Reply:
Its kind of funny reading these now, its clear I wrote this stuff while bleary-eyed at 5am lol. Merge away